app/controllers/student_requests_controller.rb

```diff
@@ -34,22 +34,21 @@ def create
     redirect_to student_requests_path
   end
   
-  def update
-    @student_request = StudentRequest.find params[:id]
-    @student_request.state = StudentRequest::WITHDRAWN_STATE
-    @student_request.save!
-    flash[:notice] = "Student Request was successfully deleted."
-    redirect_to student_requests_path
-  end
-  
   def edit
     @student_request = StudentRequest.find params[:id]
     @student_request.state = StudentRequest::WITHDRAWN_STATE
     @student_request.save!
     flash[:notice] = "Student Request was successfully deleted."
     redirect_to student_requests_path
+    
   end
 
+  def update
```

Comment:
@adityangud yaar I think u changed the definition of update method by mistake .. can u just revert the change on controller file?